movement/transforms.py

```diff
@@ -113,3 +116,155 @@ def scale(
     elif space_unit is None:
         scaled_data.attrs.pop("space_unit", None)
     return scaled_data
+
+
+def get_homography_transform(
+    srcPoints: np.ndarray, dstPoints: np.ndarray
+) -> np.ndarray:
+    """Compute a homography transformation matrix.
+
+    Parameters
+    ----------
+    srcPoints : np.ndarray
+        An array of shape (N, 2) representing N source points
+        in 2-dimensional space. N >= 4.
+        The data type is assumed to be float32.
```

Comment:
I wouldn't saay "The data type is assumed to be float32". It's rather that your code coerces it into float32. I would probably just omit this sentence (for both arguments).